### DISCUSSION TITLE
Automate writing to external DB

### DISCUSSION THREAD

**Discussion by aramay1** [Budibase/budibase#1403] [Help]
Hi under automation I can select a trigger (when a row is added) then action (create a row) what is the point of this action if it only serves the internal DB why can I select any of the DB's I have linked externally. 

Is there a workaround? 

I would like to write directly to MySQL without having to write a query for each form. can I just copy the data being written to the internal DB?  

The webhooks only allow for 4 data points to passed which does not seem like much data, if that is the intended write method would you please supply more info on how to use them. 

I appreciate your feedback.

**Comment by shogunpurple** [comment]
Hi @aramay1 - being able to execute external queries as part of automations is on our roadmap, and will be completed over the next number of months. In the meantime, the best way to do this would be to possibly trigger a zapier or integromat integration, that writes the data from the row into MySQL.

**Reply by KernOil** [reply]
Hi @shogunpurple thank you for your reply I greatly appreciate it. could you please show me or give me more info on how to properly configure the webhooks in budibase when I try and send a webhook no info is sent. there is little to no info on what the input values are and what variables can be used to get out the newly created row.
I would greatly appreciate you help.

**Reply by tasneemimamkhan** [reply]
I too would like to know! Thank you!

**Reply by shogunpurple** [reply]
No problem! I'll put together a loom video here showing how to configure a webhook to integromat and post the link.

**Reply by shogunpurple** [reply]
Guys - it looks like we are having some issues with automations and there's a bug in the trigger. I will look into this and keep you updated. The automation triggers do not seem to be firing properly when saving rows in the current release.

**Reply by tasneemimamkhan** [reply]
Thank you for letting us know!
